torch/onnx/_internal/fx/passes/fx_to_onnxscript.py

```diff
@@ -163,13 +163,26 @@ def _fill_tensor_meta(
     expected_values: Union[torch.Tensor, Tuple[torch.Tensor, ...]],
 ):
     """Fill the meta information of onnxscript_values with that from the fx FakeTensor."""
+
+    if isinstance(expected_values, (list, tuple)) and not isinstance(
+        onnxscript_values, (list, tuple)
+    ):
+        # TODO(titaiwang): Support List of Tensors: https://github.com/microsoft/onnx-script/issues/481
+        # This is the case that fx has [tensor, tensor, ...], but onnxscript_value wrapped it as a single tensor.
+        # graph_buiding not yet support list of tensors, so we don't need to handle this case for now.
+        # eg: aten_split
+        return
+
     flat_onnxscript_values, _ = _pytree.tree_flatten(onnxscript_values)
     flat_expected_values, _ = _pytree.tree_flatten(expected_values)
     for i, (onnxscript_value, expected_value) in enumerate(
         zip(flat_onnxscript_values, flat_expected_values)
     ):
         # Only set shape for now as we don't need type information.
         onnxscript_value.shape = tuple(expected_value.size())
+        # FIXME(titaiwang): type promotion is not supported yet,
+        # so set dtype will break onnx shape inference.
```

Comment:
Are we postponing the error?

Reply:
when type_promotion is provided, this will be fixed. Currently, it's blocked by aten_add. We could add `CastLike(self, other)` to pass CI test which would be a workaround.

Reply:
dtype enabled or not is not blocking anything though. It's nice to have.

Reply:
I intend to enable it in the next PR with dynamic support.

Reply:
@BowenBao It's enabled as type_promotion error is ignored https://github.com/microsoft/onnx-script/blob/ccf55862b55ccc4ecff4b8d4cc447c8db931804f/onnxscript/function_libs/torch_aten/graph_building.py#L488

So far, ORT is happy with it.